.travis.yml

```diff
@@ -3,17 +3,28 @@ language: php
 
 php:
   - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
 
-env:
-  - SYMFONY_VERSION: 2.3.*
-  - SYMFONY_VERSION: 2.4.*
-  - SYMFONY_VERSION: 2.5.*
-  - SYMFONY_VERSION: 2.6.*
-  - SYMFONY_VERSION: 2.7.*@dev
+matrix:
+  allow_failures:
+    - php: 7.0
+  include:
+    - php: 5.6
+      env: SYMFONY_VERSION="2.3.*"
+    - php: 5.6
+      env: SYMFONY_VERSION="2.4.*"
+    - php: 5.6
+      env: SYMFONY_VERSION="2.5.*"
+    - php: 5.6
+      env: SYMFONY_VERSION="2.6.*"
+    - php: 5.6
+      env: SYMFONY_VERSION="2.7.*@dev"
 
 before_install:
   - composer selfupdate
-  - composer require --no-update symfony/http-kernel:${SYMFONY_VERSION} symfony/dependency-injection:${SYMFONY_VERSION} symfony/yaml:${SYMFONY_VERSION}
+  - sh -c 'if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/http-kernel:${SYMFONY_VERSION} symfony/dependency-injection:${SYMFONY_VERSION} symfony/yaml:${SYMFONY_VERSION}; fi;'
```

Comment:
Honestly, I was not able to reproduce your issue (see xabbuh/SymfonyBridge@e631b9797be8de71853a13cbc7f39a795f5c5a00 and its [Travis build](https://travis-ci.org/xabbuh/SymfonyBridge/builds/54621952)).